docs/en/engines/table-engines/special/memory.md

```diff
@@ -6,6 +6,13 @@ sidebar_label:  Memory
 
 # Memory Table Engine
 
+:::note
+When using the Memory table engine on ClickHouse Cloud, data is not replicated across all nodes (by design). To guarantee that all queries are routed to the same node and that the Memory table engine works as expected, you can do one of the following:
+- Execute all operations in the same session
+- Use a client that uses TCP or the native interface (which enables support for sticky connections) such as [clickhouse-client](/en/interfaces/cli)
+- Submit and execute all queries at once using a multi-statement query (required with clients using the HTTP interface such as [clickhouse-connect](/en/integrations/python))
```

Comment:
I'm not sure, are multi-statements supported for HTTP? I thought the opposite.

Reply:
You're right, I misinterpreted. I can remove the last bullet point.